app/Controllers/authController.php

```diff
@@ -160,7 +160,7 @@ public function formLoginAction() {
 				Minz_Translate::init(FreshRSS_Context::$user_conf->language);
 
 				// All is good, go back to the index.
-				Minz_Request::good(_t('feedback.auth.login.success'), [ 'c' => 'index', 'a' => 'index' ]);
+				Minz_Request::good(_t('feedback.auth.login.success'), Minz_Url::unserialize(Minz_Request::param('original_request')));
```

Comment:
What if we land on the login form to start with? Risk of infinite loop?

Reply:
You're right. I've removed the original request if it's an auth request.